InvenTree/config_template.yaml

```diff
@@ -58,6 +58,14 @@ database:
 # Use the environment variable INVENTREE_DEBUG
 debug: True
 
+# Set to True to enable the admin interface
```

Comment:
Suggestion: Also mention that is enabled by default

Reply:
Done